src/appkit.cs

```diff
@@ -7750,6 +7776,27 @@ interface NSEvent : NSCoding, NSCopying {
 		[Mac (10,12,2)]
 		[Export ("coalescedTouchesForTouch:")]
 		NSTouch[] GetCoalescedTouches (NSTouch touch);
+
+		[Mac (10,15)]
+		[Export ("charactersByApplyingModifiers:")]
+		[return: NullAllowed]
+		string GetCharacters (NSEventModifierFlags modifiers);
```

Comment:
a bit weird to have `GetCharacters` returns a `string`... but it matches ObjC and I don't have a better suggestion